src/greyVest/Grid.js

```diff
@@ -1,14 +1,35 @@
 import React from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
 
-let Grid = ({ style, columns, gap, ...props }) => (
-  <div
+let formatAreas = _.flow(
+  _.map(F.quote),
+  _.join(' ')
+)
+
+let repeatNumber = F.when(_.isNumber, x => `repeat(${x}, 1fr)`)
+
+let Grid = ({
+  as: Component = 'div',
+  columns,
+  rows,
+  areas,
+  gap,
+  placeContent,
+  placeItems,
+  inline = false,
+  style,
+  ...props
+}) => (
+  <Component
     style={{
-      display: 'grid',
-      ...(columns && {
-        gridTemplateColumns: columns,
-        msGridTemplateColumns: columns,
```

Comment:
If we remove this, we will lose whatever basic IE grid support we have. I’m fine with that, but we should at least note it in the changelog